### ISSUE TITLE
Focus page from dev-tools, while leaving dev-tools opened

### ISSUE THREAD

**Issue by rlabrovi** [philc/vimium#2556]
Is there any way to focus page from developer tools and vice versa, while leaving developer tools opened.

**Comment by gdh1995** [comment]
No. Chrome hasn't provide any APIs to do such actions until now.

There's a proposal for WebExtension to allow an extension to move focus from browser address bar to web pages: https://bugzilla.mozilla.org/show_bug.cgi?id=1341416 , so if you think your feature is useful enough, you may add a new bug report on the site to try to let developers implement it.
